integration-tests/src/main/java/org/testng/TestNG.java

```diff
@@ -329,17 +329,8 @@ public void initializeSuitesAndJarFile()
               s.getChildSuites().add(cSuite);
             }
           }
-          catch (FileNotFoundException e) {
-            e.printStackTrace(System.out);
-          }
-          catch (ParserConfigurationException e) {
-            e.printStackTrace(System.out);
-          }
-          catch (SAXException e) {
-            e.printStackTrace(System.out);
-          }
-          catch (IOException e) {
-            e.printStackTrace(System.out);
+          catch (ParserConfigurationException | SAXException | IOException e) {
```

Comment:
`Exception.printStackTrace()` is also a thing that is forbidden in forbidden-apis, so had to make this change to make the project compile